model/internal/pdata/common.go

```diff
@@ -28,235 +28,235 @@ import (
 	otlpcommon "go.opentelemetry.io/collector/model/internal/data/protogen/common/v1"
 )
 
-// AttributeValueType specifies the type of AttributeValue.
-type AttributeValueType int32
+// AnyValueType specifies the type of AnyValue.
+type AnyValueType int32
 
 const (
-	AttributeValueTypeEmpty AttributeValueType = iota
-	AttributeValueTypeString
-	AttributeValueTypeInt
-	AttributeValueTypeDouble
-	AttributeValueTypeBool
-	AttributeValueTypeMap
-	AttributeValueTypeArray
-	AttributeValueTypeBytes
+	AnyValueTypeEmpty AnyValueType = iota
+	AnyValueTypeString
+	AnyValueTypeInt
+	AnyValueTypeDouble
+	AnyValueTypeBool
+	AnyValueTypeMap
+	AnyValueTypeArray
+	AnyValueTypeBytes
 )
 
-// String returns the string representation of the AttributeValueType.
-func (avt AttributeValueType) String() string {
+// String returns the string representation of the AnyValueType.
+func (avt AnyValueType) String() string {
 	switch avt {
-	case AttributeValueTypeEmpty:
+	case AnyValueTypeEmpty:
 		return "EMPTY"
-	case AttributeValueTypeString:
+	case AnyValueTypeString:
 		return "STRING"
-	case AttributeValueTypeBool:
+	case AnyValueTypeBool:
 		return "BOOL"
-	case AttributeValueTypeInt:
+	case AnyValueTypeInt:
 		return "INT"
-	case AttributeValueTypeDouble:
+	case AnyValueTypeDouble:
 		return "DOUBLE"
-	case AttributeValueTypeMap:
+	case AnyValueTypeMap:
 		return "MAP"
-	case AttributeValueTypeArray:
+	case AnyValueTypeArray:
 		return "ARRAY"
-	case AttributeValueTypeBytes:
+	case AnyValueTypeBytes:
 		return "BYTES"
 	}
 	return ""
 }
 
-// AttributeValue is a mutable cell containing the value of an attribute. Typically used in AttributeMap.
+// AnyValue is a mutable cell containing the value of an attribute. Typically used in AttributeMap.
 // Must use one of NewAttributeValue+ functions below to create new instances.
 //
 // Intended to be passed by value since internally it is just a pointer to actual
 // value representation. For the same reason passing by value and calling setters
 // will modify the original, e.g.:
 //
-//   func f1(val AttributeValue) { val.SetIntVal(234) }
+//   func f1(val AnyValue) { val.SetIntVal(234) }
 //   func f2() {
-//       v := NewAttributeValueString("a string")
+//       v := NewAnyValueString("a string")
 //       f1(v)
-//       _ := v.Type() // this will return AttributeValueTypeInt
+//       _ := v.Type() // this will return AnyValueTypeInt
 //   }
 //
-// Important: zero-initialized instance is not valid for use. All AttributeValue functions below must
+// Important: zero-initialized instance is not valid for use. All AnyValue functions below must
 // be called only on instances that are created via NewAttributeValue+ functions.
```

Comment:
Comment need updating
```suggestion
// be called only on instances that are created via NewAnyValue+ functions.
```